### ISSUE TITLE
Can I use H2O itself for websocket app development?

### ISSUE THREAD

**Issue by remoteportal** [h2o/h2o#952]
I'm hoping that H2O is the most massively scalable "C" language solution for websockets, since it's faster than libuv and has wslay built-in.

I've researched Erlang but am thinking that for my needs that H2O would scale higher since it's so darn optimized and simple.

**Comment by kazuho** [comment]
> Can I use H2O itself for websocket app development?

Yes. examples/libh2o/websocket.c is an example of a websocket server implemented using libh2o.

**Comment by narenkn** [comment]
Could someone provide a client example ?? Preferably with security "wss://" ...
Thanks!!

**Comment by yannick** [comment]
@narenkn for writing clients in C you should use any websocket library, e.g. wslay.
see https://github.com/tatsuhiro-t/wslay/blob/master/examples/testclient.cc for an example.

**Comment by narenkn** [comment]
Thank you ... will look at it..
-Naren.

**Comment by unisqu** [comment]
how do i run https://github.com/tatsuhiro-t/wslay/blob/master/examples/testclient.cc inside h2o?
